### PR TITLE
Update infer bundle class to create deepedit infer class

### PR DESCRIPTION
**Pull request by diazandr3s**
Signed-off-by: Andres <diazandr3s@gmail.com>

### CODE REVIEW

File: sample-apps/monaibundle/main.py [hunk 1]
```diff
@@ -58,10 +58,22 @@ def init_infers(self) -> Dict[str, InferTask]:
...
-            i = BundleInferTask(b, self.conf)
-            logger.info(f"+++ Adding Inferer:: {n} => {i}")
-            infers[n] = i
+            if "deepedit" in n:
+                # Adding automatic inferer
+                i = BundleInferTask(b, self.conf, type="segmentation")
+                logger.info(f"+++ Adding Inferer:: {n}_seg => {i}")
+                infers[n + "_seg"] = i
+                # Adding inferer for managing clicks
+                i = BundleInferTask(b, self.conf, type="deepedit", deepedit=True)
```
Comment (SachidanandAlle) [inline sample-apps/monaibundle/main.py:69]: you already have type="deepedit" then why you want to send deepedit=True again

Comment (diazandr3s) [reply sample-apps/monaibundle/main.py]: Absolutely right. PR updated. @SachidanandAlle

**Comment by SachidanandAlle** [comment]
Do we need this change?

**Comment by diazandr3s** [comment]
> Do we need this change?

This PR allows this DeepEdit bundle (https://github.com/Project-MONAI/model-zoo/tree/dev/models/spleen_deepedit_annotation) to be interactive in MONAI Label. Without this change, this DeepEdit model only works as an automatic model

**Review by SachidanandAlle — APPROVED** [review]

**Pull request merged**